### PR TITLE
Unit test measurement block #56:

### PR DESCRIPTION
**Pull request by jordan-heemskerk**
- Check that normals are zero or 1
- Check that vertices and normals are as expected from a small test set

Conflicts:
    src/test/kinect_fusion_opencl_measurement_pipeline_block.cpp

### CODE REVIEW

File: src/test/kinect_fusion_opencl_measurement_pipeline_block.cpp [hunk 1]
```diff
@@ -51,15 +52,92 @@ SCENARIO_METHOD(fixture, "A dynfu::kinect_fusion_opencl_measurement_pipeline_blo
...
+			auto && n=std::get<1>(t);
+			auto && vs=v->get();
+			auto && ns=n->get();
 			
-			THEN("stuff happens!") {
+			THEN("The returned normals are all either exactly 0 or close to 1 in length") {
+				
+				for(auto && n : ns) {
+				
+					CHECK((n.norm() == Approx( 1.0f ) || n.isZero() == 1));
```
Comment (RobertLeahy) [inline src/test/kinect_fusion_opencl_measurement_pipeline_block.cpp]: What's the deal with `n.isZero() == 1`?

Comment (jordan-heemskerk) [inline src/test/kinect_fusion_opencl_measurement_pipeline_block.cpp]: Leftover stuff from debugging... I'll remove it as n.isZero() returns 0 or 1

**Comment by jordan-heemskerk** [comment]
@RobertLeahy are you happy with this now?

**Comment by RobertLeahy** [comment]
:+1:

**Pull request merged**